tests/integration/modules/molecule/sensu_go_check/playbook.yml

```diff
@@ -67,19 +70,19 @@
         env_vars:
           foo: bar
         runtime_assets: awesomeness
```

Comment:
Wait what, does this `awesomeness` asset really exist? If not, this then means we just add dummy values to our integration tests without actually creating the asset :) Besides, I thought it has to be array

Reply:
I don't think sensu API is checking asset existence when creating check. I just tried it locally with handler, it accepts anything you put there, but it will fail when the check is executed.

ansible list parameter also allows a single value (I also didn't know that), this is how they handle it :D

```
    if isinstance(value, list):
        return value

    if isinstance(value, string_types):
        return value.split(",")
    elif isinstance(value, int) or isinstance(value, float):
        return [str(value)]

    raise TypeError('%s cannot be converted to a list' % type(value))
```